charts/ingress-nginx/templates/controller-serviceaccount-secret.yaml

```diff
@@ -0,0 +1,18 @@
+{{- if or .Values.serviceAccount.create -}}
+{{- if semverCompare ">=1.24.0-0" .Capabilities.KubeVersion.Version -}}
```

Comment:
Can we write them on one line?

If written separately, then we should not need the `or` function

Reply:
I'm actually removing this, it is not necessary :)

The Token / Secret is created inside the Pod anyway. Will do another commit